### ISSUE TITLE
MSI 64-bit installer for 3.12 contains unsigned executables

### ISSUE THREAD

**Issue by mroek** [sqlitebrowser/sqlitebrowser#2401]
Details for the issue
--------------------

#### What did you do?
Installed the latest official version (3.12) from the 64-bit MSI installer

#### What did you expect to see?
Executable files in the install directory to be digitally signed

#### What did you see instead?
Executable files are not digitally signed

Useful extra information
-------------------------
The 64-bit ZIP version contains digitally signed executables, so it appears the wrong files where used when the MSI installer was built.

Files were downloaded from here:
https://sqlitebrowser.org/dl/#windows

**Comment by justinclift** [comment]
Interesting.  @MKleusberg Any thoughts?

**Comment by justinclift** [comment]
@mroek Oh, hang on a second.  You're meaning the msi installer itself was signed ok, but the .exe's themselves (once installed) aren't?

If that's the case, it's a valid point.  Not sure how we should go about fixing that.  We'd probably need to repack the .msi using the signed executables.  One of our more Windows clueful people will probably have ideas. :smile:

**Comment by mroek** [comment]
@justinclift Yes, it's the executables contained within the installers that are different (and unsigned for the MSI installer).

**Comment by justinclift** [comment]
Cool.  Thanks for pointing out the problem, that'll be something we should take care of in the ~near-ish future.  Probably for the 3.12.1 release, which isn't too far off. :smile:

**Comment by MKleusberg** [comment]
Yeah, agreed. Looking forward to any ideas on how to repack the MSI files in the most easy way as I don't have a Windows system with a full dev environment available :smile:

**Comment by justinclift** [comment]
Yeah, we'll need to regenerate the .msi files somehow.  @karim Any ideas?
